knt-backend/app/models.py

```diff
@@ -2,42 +2,98 @@
 from flask_restful import Resource
 from flask import request
 
+""" USERS """
+
+
 class User(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     first_name = db.Column(db.Text)
```

Comment:
Nitpick maybe? But I think it would be easier and better to just have a name field, and the board member who adds the user in the database writes a preferred name picked with the help of the user.

https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/

Reply:
I basically put fields that i know exist in the current K&T admin tool - it has a separate field for first, last and nickname.

Reply:
Good point, we will deal with this later (famous last words?)